docs/setup.md

```diff
@@ -66,6 +70,24 @@ Usage details:
 | `-r / --ref` | Path to reference fasta. Mutually exclusive with -a|
 | `-a / --acc` | NCBI accession of reference. Mutually exclusive with -r|
 
+#### Handling samples with more than one pair of reads
+
+In order to specify samples that were sequenced multiple times in your sample sheet, you must:
+1. Create a duplicate row for each unit of sequencing
+2. Ensure the `BioSample` value is the same across all rows for the sample.
+3. Give each row a unique `Run` value. This allows snpArcher to collect all alignments for a `BioSample`.
+4. Give each row a unique `LibraryName` value, if applicable. Used for marking duplicates, `LibraryName` should be the same in cases where the same library prep was sequenced across multiple lanes. See [here](https://gatk.broadinstitute.org/hc/en-us/articles/360035890671-Read-groups) for more info. 
```

Comment:
replace "library prep" with "library". At the end, I think you could add "Note that the sample sample, but prepared with different library protocols, should have a unique LibraryName for each preparation"